cli/account/forms.py

```diff
@@ -21,29 +14,5 @@ class GetAccountBalanceForm(Schema):
     Get balance of the account form.
     """
 
-    address = fields.String(required=True)
-    node_url = fields.String(allow_none=True, required=False)
-
-    @validates('address')
-    def validate_address(self, address):
-        """
-        Validate account address.
-        """
-        if re.match(pattern=ADDRESS_REGEXP, string=address) is None:
-            raise ValidationError(f'The following address `{address}` is invalid.')
-
-    @validates('node_url')
-    def validate_node_url(self, node_url):
-        """
-        Validate node URL.
-
-        If node URL is localhost, it means client didn't passed any URL, so nothing to validate.
-        """
-        if node_url == 'localhost':
-            return
-
-        if 'http' in node_url or 'https' in node_url:
-            raise ValidationError(f'Pass the following node URL `{node_url}` without protocol (http, https, etc.).')
-
-        if re.match(pattern=DOMAIN_NAME_REGEXP, string=node_url) is None:
-            raise ValidationError(f'The following node URL `{node_url}` is invalid.')
+    address = AccountAddressField(required=True)
```

Comment:
This is an extra check.
```suggestion
    address = AccountAddressField(required=False)
```
Because of [click](https://github.com/Remmeauth/remme-core-cli/blob/develop/cli/account/cli.py#L34).

Reply:
@Alladin9393, imagine we will migrate this code from CLI to Web. Do we need to pay additional attention to how CLI handle arguments? No, we don't. We just need to cut the CLI. I mean forms do not know about the CLI.

Reply:
@dmytrostriletskyi, ok.